.github/workflows/test.yml

```diff
@@ -309,7 +309,9 @@ jobs:
             tags: parabricks/haplotypecaller
           - profile: conda
             tags: parabricks/indexgvcf
-          - profile: conda
+          - profile: "conda"
+            tags: parabricks/dbsnp
```

Comment:
can we have that properly sorted?

Reply:
Right, I've just fixed it. Still having troubles with pytest though